R/rdeephaven/R/update_by_ops_wrapper.R

```diff
@@ -14,76 +95,209 @@ UpdateByOp <- R6Class("UpdateByOp",
   )
 )
 
-### All of the functions below return an instance of an 'UpdateByOp' object
 
+#' @name
+#' uby_cum_sum
+#' @title
+#' Cumulative sum of specified columns by group
+#' @md
+#'
 #' @description
 #' Creates a cumulative sum UpdateByOp that computes the cumulative sum of each column in `cols` for each aggregation group.
+#'
+#' @details
+#' The aggregation groups that this function acts on are defined with the `by` parameter of the `update_by()` caller
+#' function. The aggregation groups are defined by the unique combinations of values in the `by` columns. For example,
+#' if `by = c("A", "B")`, then the aggregation groups are defined by the unique combinations of values in the
+#' `A` and `B` columns.
+#'
+#' This function, like the other Deephaven `uby_*` functions, is a generator function. That is, its output is another
+#' function that is intended to be used in a call to `update_by()`. This detail is typically hidden from the user by
+#' `update_by()`, which calls the generated functions internally. However, it is important to understand this detail
+#' for debugging purposes, as the output of a `uby_*` function can otherwise seem unexpected.
+#'
 #' @param cols String or list of strings denoting the column(s) to operate on. Can be renaming expressions, i.e. “new_col = col”.
 #' Default is to compute the cumulative sum for all non-grouping columns.
 #' @return UpdateByOp to be used in `update_by()`.
+#'
+#' @examples
+#' print("hello!")
```

Comment:
`print("hello!")` ?  Is that intentional?  :-)

Reply:
There is some amount of repetition in the doc strings below for other functions like this one, so two comments:
One is to try to think about some way to try to reduce that repetition.  You could point to a common part of the documentation with a note, instead of copy&paste.  That is not necessarily "obviously better", but I think it is worth to think about doing it;  see if you feel one way or the other is better.  The issue with duplication is maintainability of us.  But is not only that, users coming to the documentation for a specific check/answer will see that comment multiple times; so it may also make sense to give them the option to not having to see the whole thing every time.